frontend/src/scenes/max/Thread.tsx

```diff
@@ -358,10 +339,9 @@ function SuccessActions({
         if (!feedback) {
             return // Input is empty
         }
-        posthog.capture('chat feedback', {
-            question: relevantHumanMessage.content,
-            answer: JSON.stringify(relevantVisualizationMessage.answer),
-            feedback,
+        posthog.capture('$ai_feedback', {
+            $ai_feedback_text: feedback,
+            $ai_trace_id: traceId,
         })
```

Comment:
We must add a `posthog-js` method for this – users should never have to type dollar-prefixed properties, because they're the features we offer out of the box. This way we ensure the SDKs and the product's UI all work together neatly, and the user has a smooth time implementing. Let's do this as a follow-up